src/toil/utils/toilStatus.py

```diff
@@ -15,47 +15,72 @@
 """
 Reports the state of a Toil workflow
 """
+
+# python 2/3 compatibility imports
 from __future__ import absolute_import
 from __future__ import print_function
+# TODO: change functions to support python 3 str and map
+# from builtins import map
+# from builtins import str
 
-from builtins import map
-from builtins import str
+# standard library
 import logging
 import sys
 
-from toil.lib.bioio import logStream
+# toil imports
 from toil.lib.bioio import getBasicOptionParser
 from toil.lib.bioio import parseBasicOptions
 from toil.common import Toil, jobStoreLocatorHelp, Config
-from toil.leader import ToilState
 from toil.job import JobException
 from toil.version import version
 
 logger = logging.getLogger( __name__ )
 
 def main():
-    """Reports the state of the toil.
```

Comment:
Wasn't sure where to put this comment... Are the changes/additions in functionality here being tested? I think the stats mechanism is tested in `test.utilsTest.UtilsTest.testUtilsSort` (and probably other places). Are more tests necessary?